src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
+    White,
+    Black,
+    Orange,
+    Green,
+    Purple,
+    Violet
```

Comment:
[Checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)
Don’t forget how to name constants according to style guide (the same with naming of enum values)